.eslintrc.js

```diff
@@ -179,6 +179,7 @@ module.exports = {
       rules: {
         '@typescript-eslint/unbound-method': 'off',
         'jest/unbound-method': 'error',
+        '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
should this be an inline ts-ignore? I worry about how tempting this will become.

Reply:
This is only for test files. Seems kinda common to use any when creating a stub object